guide/additional-info/changes-in-v12.md

```diff
@@ -1027,17 +1468,27 @@ guild.createChannel('Secret Voice Channel', 'voice', {
 
 ### GuildChannel
 
-#### GuildChannel#lockPermissions
+#### GuildChannel#permissionsLocked
 
-`channel.lockPermimssions()` has been added.
+`guildChannel.permissionsLocked` is a boolean value representing if the `permissionOverwrites` of the channel match its parents' `permissionOverwrites`.
 
-#### GuildChannel#parent(ID)
+#### GuildChannel#updateOverwrites
 
-`channel.parent` and `channel.parentID` have been added.
+Creates or update an existing overwrite for a user or role.  The first parameter is a `PermissionOverwriteOption` object; the second, optional parameter is `reason`, a string.
 
-#### GuildChannel#setParent
+```js
+channel.updateOverwrite(message.author, {
+	SEND_MESSAGES: false
```

Comment:
Missing a trailing comma

Reply:
What trailing comma?  Not seeing where it's missing one.

Reply:
After the `SEND_MESSAGES: false` there should be a trailing comma. This is also the line that's failing the circleci linting.
```js
channel.updateOverwrite(message.author, {
	SEND_MESSAGES: false,
});
```

Reply:
Apparently Sanc likes having linting rules that aren't consistent with the actual docs.  Trailing commas were added, as well as a few other typos fixed.  Thank you for pointing them out.

Reply:
This is not related to the docs at all, you should always follow linting rules of the actuall project and nothing else. If sanc decides to have these and the official docs dont have that is completely okay and people contributing should follow the local ones.

Reply:
Yeah, I just didn't expect them to be different from the docs, I just grabbed the examples from there and thought it'd be ok.  Wasn't aware there'd be a conflict and I'm still learning git and linting and all that, it's all good.